ruff/__main__.py

```diff
@@ -4,4 +4,4 @@
 
 if __name__ == "__main__":
     ruff = os.path.join(sysconfig.get_path("scripts"), "ruff")
-    os.spawnv(os.P_WAIT, ruff, [ruff, *sys.argv[1:]])
+    exit(os.spawnv(os.P_WAIT, ruff, [ruff, *sys.argv[1:]]))
```

Comment:
`exit()` is not a built-in function and is only defined if the site module is imported during Python's initialization (which, for example, can be disabled with `-S` option passed to the interpreter). You should use `sys.exit()` instead:
```suggestion
    sys.exit(os.spawnv(os.P_WAIT, ruff, [ruff, *sys.argv[1:]]))
```

Reply:
Fixed, thanks. (Ruff should have a rule for this!)